ui/src/components/TrafficPage.tsx

```diff
@@ -100,16 +101,30 @@ export const TrafficPage: React.FC<TrafficPageProps> = ({setAnalyzeStatus, onTLS
 
     const listEntry = useRef(null);
 
-    const openWebSocket = (query) => {
-        setFocusedEntryId(null);
-        setEntries([]);
-        setEntriesBuffer([]);
+    const openWebSocket = (query: string, dontClear: boolean) => {
+        if (!dontClear) {
```

Comment:
maybe another name is better, it was hard for e to understand (especially when the name is negative name).. 
I prefer positive names which makes it easier when reading (no need to think about `!dont*` expressions)
about the name itself - maybe `resetEntriesBuffer` or something like this is better?

Reply:
@gadotroee makes sense. Done with https://github.com/up9inc/mizu/commit/19157139d9c901f18755e132f1083f99a8c0fd6b